packages/SwingSet/src/controller/controller.js

```diff
@@ -33,6 +34,12 @@ import {
 import { makeStartXSnap } from './startXSnap.js';
 import { makeStartSubprocessWorkerNode } from './startNodeSubprocess.js';
 
+const endoZipBase64Sha512Pattern = harden({
```

Comment:
I think we prefer "Shape", cf. https://github.com/endojs/endo/blob/643a2fb187604e76adbbeb8e9d19fb195a58dd79/packages/patterns/src/types.js#L80-L86
```suggestion
const endoZipBase64Sha512Shape = harden({
```